src/LogicTasks/Semantics/Resolve.hs

```diff
@@ -47,7 +48,7 @@ third3 (_,_,c) = c
 genResInst :: ResolutionConfig -> Gen ResolutionInst
 genResInst ResolutionConfig{ baseConf = BaseConfig{..}, ..} = do
   clauses <- inst
-  pure $ ResolutionInst clauses printSolution extraText
+  pure $ ResolutionInst clauses printFeedbackImmediately printSolution extraText
```

Comment:
Probably better to use record matching now, to not be dependent on the order in which the two `Bool` fields appear in the type. (Imagine the order is changed in the type definition. The compiler will not be able to catch a regression here.)